app/views/idv/agreement/show.html.erb

```diff
@@ -0,0 +1,55 @@
+<% title t('doc_auth.headings.lets_go') %>
+
+<%= render AlertComponent.new(
+      type: :error,
+      class: [
+        'js-consent-form-alert',
+        'margin-bottom-4',
+        flow_session[:error_message].blank? && 'display-none',
+      ].select(&:present?),
+      message: flow_session[:error_message].presence || t('errors.doc_auth.consent_form'),
+    ) %>
+
+<%= render PageHeadingComponent.new.with_content(t('doc_auth.headings.lets_go')) %>
+<p><%= t('doc_auth.info.lets_go') %></p>
+<h2><%= t('doc_auth.headings.verify_identity') %></h2>
+<p><%= t('doc_auth.info.verify_identity') %></p>
+<h2><%= t('doc_auth.headings.secure_account') %></h2>
+<p><%= t('doc_auth.info.secure_account') %></p>
+
+<%= simple_form_for(
+      :doc_auth,
+      url: url_for,
+      method: 'put',
+      html: { autocomplete: 'off', class: 'margin-top-2 margin-bottom-5 js-consent-continue-form' },
+    ) do |f| %>
+  <%= render ClickObserverComponent.new(event_name: 'IdV: consent checkbox toggled') do %>
+    <%= render ValidatedFieldComponent.new(
+          form: f,
+          name: :ial2_consent_given,
+          as: :boolean,
+          label: t('doc_auth.instructions.consent', app_name: APP_NAME),
+          required: true,
+        ) %>
+  <% end %>
+  <p class="margin-top-2">
+    <%= new_tab_link_to(
+          t('doc_auth.instructions.learn_more'),
+          policy_redirect_url(flow: :idv, step: :agreement, location: :consent),
+        ) %>
+  </p>
+  <div class="margin-top-4">
+    <%= render(
+          SpinnerButtonComponent.new(
+            type: :submit,
+            big: true,
+            wide: true,
+            spin_on_click: false,
+          ).with_content(t('doc_auth.buttons.continue')),
+        ) %>
+  </div>
+<% end %>
+
+<%= render 'idv/doc_auth/cancel', step: 'agreement' %>
+
+<%= javascript_packs_tag_once('document-capture-welcome') %>
```

Comment:
Not at all related to the scope of the ticket, but it occurs to me that `document-capture-welcome`'s sole job is to add the `skip_upload` field, which is only used for the agreement step. So the script probably ought to be removed from the welcome step and renamed accordingly.

Reply:
I was eyeing that too. Thanks for confirming that it's now unused for welcome step. I'll rename either in this PR or separately.

Reply:
Added a note about renaming `document-capture-welcome` to [LG-9367](https://cm-jira.usa.gov/browse/LG-9367), because I'd rather make changes after this is fully deployed and the old AgreementStep is deleted.